docs/publishing/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Publishing Web Components
+section: guides
+tags:
+  - guides
+  - guide
+---
+
+## Publishing Web Components
+
+If you have developed a package that you would like to make available to other developers via a public registry, take the following concepts into account to ensure that external users of your code may make the most of it in their projects. If you've made it this far, your package will likely already include things like [demos](/demoing/), [documentation](/demoing/storybook-addon-markdown-docs.html), [tests](/testing/), et al. Each of these play a role in making the clearest example of the benefits on your work, outlining the easiest path to integrating that work into other projects, and ensuring that the package as a whole is resilient to change over time, but are not discussed directly below. If you're looking for information on including these things in your published packages, take a look at our [guides](/guide/index.html) or learn about [getting started](#getting-started) below.
```

Comment:
- "via a public registry"
maybe just be explicit and say NPM here? A public registry seems needlessly unspecific
- "Each of these play a role in making the clearest example of the benefits on your work, outlining the easiest path to integrating that work into other projects, and ensuring that the package as a whole is resilient to change over time, but are not discussed directly below." 
this sentence is a bit long and by the end of it im slightly confused, maybe chop this up a bit?

Reply:
Do you think we should name the page "Publishing Web Components to NPM"? I added "via a public registry" because I thought it would allow for the vaguery of publishing to places like GitHub Package Registry, but maybe specificity is better, top to bottom?

Good point on the following sentence!

Reply:
Hm yeah I see your point, but as far as I can tell the github registry hasnt really picked up steam or a lot of usage, especially now that npm is part of github I guess. I also feel like mentioning NPM will probably help with searchability, but thats a gut feeling. Im assuming people might be searching for "how to publish web component npm" (or thats what id google anyway)